File: MORM.Java/src/MORM/Java/Models/Venda/Produto.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "PRODUTO")
public class Produto extends CollectionItem
{
    @CampoAttribute(Campo = "ID_PRODUTO", Tipo = CampoTipo.Key)
    public String Id_Produto;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "CD_PRODUTO", Tipo = CampoTipo.Req)
    public Integer Cd_Produto;
    @CampoAttribute(Campo = "DS_PRODUTO", Tipo = CampoTipo.Req)
    public String Ds_Produto;
    @CampoAttribute(Campo = "CD_ESPECIE", Tipo = CampoTipo.Req)
    public String Cd_Especie;
    @CampoAttribute(Campo = "CD_NCM", Tipo = CampoTipo.Req)
    public String Cd_Ncm;
    @CampoAttribute(Campo = "CD_CST", Tipo = CampoTipo.Req)
    public String Cd_Cst;
    @CampoAttribute(Campo = "CD_CSOSN", Tipo = CampoTipo.Req)
    public String Cd_Csosn;
    @CampoAttribute(Campo = "PR_ALIQUOTA", Tipo = CampoTipo.Req)
    public Float Pr_Aliquota;
    @CampoAttribute(Campo = "TP_PRODUCAO", Tipo = CampoTipo.Req)
    public Integer Tp_Producao;
    @CampoAttribute(Campo = "VL_CUSTO", Tipo = CampoTipo.Req)
    public Float Vl_Custo;
    @CampoAttribute(Campo = "VL_VENDA", Tipo = CampoTipo.Req)
    public Float Vl_Venda;
    @CampoAttribute(Campo = "VL_PROMOCAO", Tipo = CampoTipo.Req)
    public Float Vl_Promocao;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Produto.cs
using MORM.Util.Atributos;
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("PRODUTO")]
    public class Produto : CollectionItem
    {
        [Campo("ID_PRODUTO", CampoTipo.Key)]
        public string Id_Produto { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("CD_PRODUTO", CampoTipo.Req)]
        public int Cd_Produto { get; set; }
        [Campo("DS_PRODUTO", CampoTipo.Req)]
        public string Ds_Produto { get; set; }
        [Campo("CD_ESPECIE", CampoTipo.Req)]
        public string Cd_Especie { get; set; }
        [Campo("CD_NCM", CampoTipo.Req)]
        public string Cd_Ncm { get; set; }
        [Campo("CD_CST", CampoTipo.Req)]
        public string Cd_Cst { get; set; }
        [Campo("CD_CSOSN", CampoTipo.Req)]
        public string Cd_Csosn { get; set; }
        [Campo("PR_ALIQUOTA", CampoTipo.Req)]
        public double Pr_Aliquota { get; set; }
        [Campo("TP_PRODUCAO", CampoTipo.Req)]
        public int Tp_Producao { get; set; }
        [Campo("VL_CUSTO", CampoTipo.Req)]
        public double Vl_Custo { get; set; }
        [Campo("VL_VENDA", CampoTipo.Req)]
        public double Vl_Venda { get; set; }
        [Campo("VL_PROMOCAO", CampoTipo.Req)]
        public double Vl_Promocao { get; set; }
    }
}